eclair-core/src/main/scala/fr/acinq/eclair/db/sqlite/SqliteAuditDb.scala

```diff
@@ -38,9 +39,9 @@ class SqliteAuditDb(sqlite: Connection) extends AuditDb with Logging {
   import ExtendedResultSet._
 
   val DB_NAME = "audit"
-  val CURRENT_VERSION = 4
+  val CURRENT_VERSION = 5
 
-  case class RelayedPart(channelId: ByteVector32, amount: MilliSatoshi, direction: String, relayType: String, timestamp: Long)
+  case class RelayedPart(channelId: ByteVector32, amount: MilliSatoshi, direction: String, relayType: String, recipientNodeId: Option[PublicKey], recipientAmount: Option[MilliSatoshi], timestamp: Long)
```

Comment:
It bothers me a bit that:

- this is only set for trampoline and `None` for channel relays
- this is duplicated when using MPP (all parts in a trampoline payment will have the same value here)

IIRC we already hesitated in the past to add this info, but chose to omit it, @pm47 do you remember why?

Reply:
I agree, I've changed the design to put all the new data in a separate table. Now there are no empty columns for channel relays and no duplication.

Reply:
There is a potential source of confusion here because now we have a `relayed` table and `relayed_trampoline` table, but both of them contain data related to trampoline payments. The `relayed` table is more granular and refers to individual in/out transfers at the channel level, that make full relays when aggregated. Can't think of a better name though.

> IIRC we already hesitated in the past to add this info, but chose to omit it, @pm47 do you remember why?

🤷‍♂️